cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraClusterInfo.java

```diff
@@ -392,12 +392,15 @@ private TokenRangeMapping<RingInstance> getTokenRangeReplicas()
         Multimap<RingInstance, Range<BigInteger>> tokenRangesByInstance;
         try
         {
+            long start = System.nanoTime();
             TokenRangeReplicasResponse response = getTokenRangesAndReplicaSets();
+            long elapsedTimeNanos = System.nanoTime() - start;
             replicaMetadata = response.replicaMetadata();
 
             tokenRangesByInstance = getTokenRangesByInstance(response.writeReplicas(), response.replicaMetadata());
-            LOGGER.info("Retrieved token ranges for {} instances from write replica set ",
-                        tokenRangesByInstance.size());
+            LOGGER.info("Retrieved token ranges for {} instances from write replica set in {} nanoseconds",
```

Comment:
this feels like a debug level log entry to me

Reply:
This is modifying an existing log entry to also publish the response time (until we start publishing metrics). 

This is useful at info level since we log at the point of invocation and before any transformation of the response.